config/locales/de.events.yml

```diff
@@ -117,11 +117,12 @@ de:
       deadline_approaching:
         one: "Bewerbungsfrist in einem Tag"
         other: "Bewerbungsfrist in %{count} Tagen"
+        today: "Bewerbungsfrist heute"
```

Comment:
Can't I18n also do `zero`? That would allow you to simplify `Event#application_deadline_label`